src/wp-includes/admin-bar.php

```diff
@@ -86,7 +86,7 @@ function wp_admin_bar_render() {
 	/**
 	 * Loads all necessary admin bar items.
 	 *
-	 * This is the hook used to add, remove, or manipulate admin bar items.
+	 * This is the hook used to add, remove, or manipulate admin bar items. The recommended hook instead of this is wp_before_admin_bar_render.
```

Comment:
```suggestion
	 * This hook can add, remove, or manipulate admin bar items. The priority
	 * determines the placement for new items, and changes to existing items
	 * would require a high priority. To remove or manipulate existing nodes
	 * without a specific priority, use `wp_before_admin_bar_render`.
```

This should be the preferred hook for adding items, and it still can modify or remove items if done with a proper priority level.

Reply:
@sabernhardt Thanks for the detailed feedback. I have implemented the suggestions, please review the same.

Reply:
Checked and the requested changes are now implemented in the PR ✅

Reply:
Thanks for updating the `admin_bar_menu` docblock.

Could you also remove the sentence about the "most optimal point" in `wp_admin_bar_render()` and limit the line length to 80 characters?
https://github.com/WordPress/wordpress-develop/blob/b50a842d4889f2e635b161f4c2d14d078651bf28/src/wp-includes/admin-bar.php#L64-L67

```
 * It includes the {@see 'admin_bar_menu'} action, which should be used to hook in and
 * add new menus to the admin bar. This also gives you access to the `$post` global,
 * among others.
```

Reply:
Done in commit https://github.com/WordPress/wordpress-develop/pull/7222/commits/cb0c76490cb7b3b7af11af7d9d06459d83488b90 ✅